samples/client/petstore/spring-http-interface-reactive/src/main/java/org/openapitools/model/SpecialModelName.java

```diff
@@ -17,7 +17,7 @@
  * SpecialModelName
  */
 
-@JsonTypeName("$special[model.name]")
+@JsonTypeName("_special_model.name_")
```

Comment:
I wouldn't expect some of these changes from your mustache changes - bad branch?